### ISSUE TITLE
Does this work?

### ISSUE THREAD

**Issue by mikestecker** [solcates/Amazon.bundle#1]
Does this work? I tried to manually install it into my plugins folder but Plex Media Server doesn't seem to see it. I am running my server on a QNAP TS-469L and my main client is running on OpenELEC-Plex

**Comment by solcates** [comment]
No, i just started it as a POC, but have since lost interest in the project :)
